schema/proto3/neighbor.proto

```diff
@@ -29,17 +29,15 @@ enum NeighborType {
 }
 
 message NeighborConfiguration {
-    uint32 format_version = 1;
-    uint32 revision_number = 2;
-
-    string request_id = 3;
-    string id = 4;
-    MessageType message_type = 5; // DELTA (default) or FULL
-    string project_id = 6;
```

Comment:
format_version and project id are all cut, right?

Reply:
yes, format_version are not useful except the top level GoalState message. We are only keeping project ID in VPC.proto per last discussion.